src/test/java/com/epam/edumanagementsystem/admin/impl/SuperAdminServiceImplTest.java

```diff
@@ -0,0 +1,62 @@
+package com.epam.edumanagementsystem.admin.impl;
+
+import com.epam.edumanagementsystem.admin.model.entity.SuperAdmin;
+import com.epam.edumanagementsystem.admin.rest.repository.SuperAdminRepository;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
+
+@SpringBootTest
+class SuperAdminServiceImplTest {
```

Comment:
During testing Service classes, where we are acting with entities or dtos, it would be better to create a dto object and test that object and its fields. For example here, you can create one SuperAdmin object in the separate method and annotate it for example with @Before annotation, i.e. it will create that object before running the tests and you can use that object in every test method of the class [https://stackoverflow.com/questions/46336915/usage-of-before-in-junit-testing].